kwanghwi/myapp/.gitignore

```
@@ -0,0 +1,2 @@
+.env
+node_modules/
```

db/schema.sql 파일이 github 상에 공유되지 않도록 추가해주세요!

```suggestion
.env
node_modules/
db/schema.sql
```